spec/serializer_spec.rb

```diff
@@ -788,7 +794,7 @@ def read_attribute_for_validation(attr)
           serialize_primary(comment_user, {serializer: MyAppOtherNamespace::UserSerializer}),
         ],
       }
-      includes = ['long-comments', 'long-comments.user']
+      includes = ['long-comments.user']
```

Comment:
Now you're testing only once include, not an array as before, and you did that everywhere.
From the specs it looks like you're changing functionality instead of extending, but that's not what you're doing.

Reply:
Exactly. Im changing it cuz it was incorrectly implemented with the standard.
Take a look here: https://github.com/fotinakis/jsonapi-serializers/pull/116

Right now including compound relations will also include all chain linkage.
Specs are in 100% correct :)

Reply:
That's a valid concern, but I think it would be best to get back to it once we have Inbox ready, this is already merged anyway and we are running out of time really. I think there are a couple of more things to be fixed in this gem 😅

Reply:
okay

Reply:
Those tests weren't for checking includes. They were intentionally duplicated to
`includes = ['long-comments', 'long-comments.user']` to include the linkage as the implementation of the standard probably changed during the gem development (cuz its referring to the topic that doesn't exist anymore).

Reply:
If there would be specif test which would test how the logic behaves with multuple elements I wouldn't change it.

Reply:
But we should also change right now the test name form 
`includes linkage in compounded resources only if the immediate parent was also included` to includes linkage in compounded resources` as this is the case right now

Reply:
yeah I got it, so we're missing a spec for multiple includes and a spec naming fix would due as well

Reply:
this gem is missing a lot of tests :D

Reply:
yay 🙌 😅

Reply:
Yes, that's not cool. but I think we will migrate from it sooner or later

Reply:
One beer says it's later! 😄 It gets the job done :P